src/pages/AdminDashboard/roomsActions.ts

```diff
@@ -36,16 +58,22 @@ export const createRoom = (t, room: Room, remindAt: number) => async (
     const api = new Api(token)
 
     try {
-        const { roomId } = await api.createRoom(room)
+        const { roomId, roomUrl } = await api.createRoom(room)
         dispatch(getRooms(t))
-        dispatch(createReminder(t, roomId, remindAt))
+
+        if (remindAt) dispatch(createReminder(t, roomId, remindAt))
```

Comment:
createRoom can also be passed 3 params to create reminders: destinations, sendsAt array, and hostName

Reply:
#477